content/ss23/master/m4-shop-gpt/future.md

```diff
@@ -3,7 +3,7 @@ title = "Future"
 weight = 5
 +++
 
-{{<section title="Multiple Images Uploaded" >}}
+{{<section title="Uploading multiple images" >}}
```

Comment:
It should be "Uploading Multiple Images" because other titles also had the first uppercase letter.